sponsorship_compassion/view/invoice_line_view.xml

```diff
@@ -8,9 +8,7 @@
             <field eval="18" name="priority"/>
             <field name="arch" type="xml">
                 <field name="name" position="after">
-                    <field name="contract_id"
-                           options="{'create_edit':false,'field_color': 'state',
-                                     'colors': {'cancelled': 'gray', 'terminated': 'gray', 'draft': 'blue', 'mandate': 'red', 'waiting': 'green'}}"/>
+                    <field name="contract_id" />

```

Comment:
Apparently you had not a good version of the file or I don't know why but this is not valid change.